apps/backend/src/modules/user/user-session.store.ts

```
@@ -1,31 +1,93 @@
+      const session = this.connectedUsers.get(socketId)
+      if (session) {
+        sessions.push(session)
+      } else {
+        // 데이터 불일치 시 정리 (방어 코드)
+        toRemove.push(socketId)
+      }
+    }
+
+    // 불일치 데이터 정리
+    if (toRemove.length > 0) {
+      toRemove.forEach(id => socketIds.delete(id))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)